L91-improved-directory-support-for-python-bazel-rules.md

```diff
@@ -0,0 +1,86 @@
+Improved handling of subdirectories and external repositories for py_proto_library
+----
+* Author(s): Thomas K&ouml;ppe (tkoeppe)
+* Approver: gnossen
+* Status: Draft
+* Implemented in: Starlark
+* Last updated: 2021-11-29
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Don't forget to create this thread. Creation of the thread is what starts the two week comment period.

Reply:
Done: https://groups.google.com/g/grpc-io/c/paqz77BJ0Js

Reply:
@gnossen: PTAL, what do you reckon?